cclient/response/get_neighbors.h

```diff
@@ -14,60 +23,121 @@
 extern "C" {
 #endif
 
+/**
+ * @brief The data structure of the neighbor object.
+ *
+ */
 typedef struct {
   /**
-   * URI of your peer.
+   * address and port number of your peer.
    */
   char_buffer_t* address;
   /**
-   * The total number of transactions which includes sent, stale, etc.
+   * Number of all transactions(Invalid, valid and already-seen) this neighbor sent to you.
    */
   uint32_t all_trans_num;
   /**
-   *
+   * Number of random tips this neighbor requested from your node.
    */
   uint32_t random_trans_req_num;
   /**
-   * Number of newly transmitted transactions.
+   * Number of new transactions received from this neighbor. New transactions are transactions that you have not yet
+   * stored when this neighbor sends it to you.
    */
   uint32_t new_trans_num;
   /**
-   * Invalid transactions your peer has sent you. These are transactions with
-   * invalid signatures or overall schema.
+   * Number of invalid transactions this neighbor sent to you. These are transactions with invalid signatures or overall
+   * schema.
    */
   uint32_t invalid_trans_num;
   /**
-   * The number of stale transactions
+   * Stale transactions this neighbor has sent you. These are transactions with a timestamp older than your latest
+   * snapshot.
    */
   uint32_t stale_trans_num;
   /**
-   * The number of sent transactions.
+   * Amount of transactions send through this neighbor.
+   * Number of all new transactions you get from other neighbors and forward to this neighbor. (You forward all new
+   * transactions from one neighbor to all other neighbors) So the number is exactly the sum of all new transactions
+   * from your neighbors except this neighbor.
```

Comment:
This is not entirely true, it also includes transactions sent from tip requesting and transaction requesting